problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,10 +1,52 @@
 \begin{exercises}
+    %45 min
+    %60 min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$, given by some matrix $M_R$. Compute $M_R$.
+	    
+	    \prob Let $\mathcal P:\R^2\to\R^2$ be projection onto the $y$-axis, given by some matrix $M_P$. Compute $M_P$.
+	    
+	    \prob Let $\mathcal F:\R^2\to\R^2$ be a transformation flipping over the line $y=x$, given by some matrix $M_F$. Compute $M_F$.
+	    
+	    \prob Let $\mathcal S:\R^2\to\R^2$ be a transformation stretching by 2, given by some matrix $M_F$. Compute $M_F$.
```

Comment:
`stretching by 2` -> `that doubles every vector`. and `M_F`->`M_{\mathcal S}`

All of these can be parts of the same question. This one should be first, since it is the easiest one.